lucene/core/src/java/org/apache/lucene/document/FieldType.java

```diff
@@ -351,6 +353,27 @@ public int pointNumBytes() {
     return dimensionNumBytes;
   }
 
+  @Override
+  public int vectorDimension() {
+    return vectorDimension;
+  }
+
+  /**
+   * Sets the vector dimension. Passing a value greater than zero will enable vector indexing.
+   */
+  public void setVectorDimension(int vectorDimension) {
+    if (vectorDimension < 0) {
```

Comment:
`vectorDimension < 0`=>  `vectorDimension <= 0`?
Looks like we don't allow 0 dimension in `VectorField`.